src/config/env.json

```diff
@@ -4,7 +4,7 @@
     "networkId": 1
   },
   "customRPC": {
-    "url": "https://rpc.tenderly.co/fork/29234e00-3487-42b2-a73b-576d3e991d05",
-    "enabled": false
+    "url": "https://rpc.tenderly.co/fork/112e7699-fa6c-4f5d-b184-10627f12235c",
+    "enabled": true
```

Comment:
needs to change

Reply:
once we're ready to merge, i'll change this.